plugins/catalog-backend/package.json

```diff
@@ -45,6 +45,7 @@
     "express-promise-router": "^3.0.3",
     "fs-extra": "^9.0.0",
     "git-url-parse": "^11.4.4",
+    "glob": "^7.1.6",
```

Comment:
We've used `globby` in other parts of the monorepo, and it comes with a promise API. Maybe worth using that instead?

Reply:
Hmm, choose `glob` as the `UrlReaderProcessor` is using `minimatch` and they are from the same author (and it uses `minimatch` internally). Globby seems to use something else under the hood, but I guess they should be compatible? But `glob` should be fine as it's already in our dependency tree.

Reply:
Cool - ok - nice DD. merging.